script.pkg_tjdashboard.php

```diff
@@ -0,0 +1,464 @@
+<?php
+/**
+ * @version    SVN: <svn_id>
+ * @package    Com_Tjlms
+ * @copyright  Copyright (C) 2005 - 2014. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ * Shika is free software. This version may have been modified pursuant
+ * to the GNU General Public License, and as distributed it includes or
+ * is derivative of works licensed under the GNU General Public License or
+ * other free or open source software licenses.
+ */
+
+// No direct access
+defined('_JEXEC') or die;
+
+jimport('joomla.filesystem.folder');
+jimport('joomla.filesystem.file');
+jimport('joomla.application.component.controller');
+
+if (!defined('DS'))
+{
+	define('DS', '/');
+}
+
+/**
+ * Tjlms Installer
+ *
+ * @since  1.0.0
+ */
+class Pkg_TjdashboardInstallerScript
+{
+	/** @var array The list of extra modules and plugins to install */
+	private $oldversion = "";
+
+	private $installation_queue = array(
+										'plugins' => array(
+															"tjdashboardrenderer" => array(
+																	'chartjs' => 1,
+																	'tabulator' => 1,
+																),
+															"api" => array(
+																	"tjdashboard" => 1
+																)
+														)
+									);
+
+	private $uninstall_queue = array(
+										'plugins' => array(
```

Comment:
@monikaatvowel Is this required? If yes then please optimize the code and formatting.

cc
@coolbung @twsvaishali